provider/cluster_rosa_classic_resource.go

```diff
@@ -1542,3 +1621,7 @@ func (r *ClusterRosaClassicResource) waitTillClusterIsNotFoundWithTimeout(ctx co
 
 	return false, nil
 }
+
+func stringParamIsNotEmpty(param types.String) bool {
```

Comment:
Can you move this into the common pkg? Seems like this function can be used almost everywhere.

Moreover, I would call it `IsStringAttributeEmpty` or something like that